lib/facts.rb

```diff
@@ -18,16 +18,17 @@ def initialize
   # Resolves fact on the system, if a specific fact is given,
   # it will only resolve that fact,
   # otherwise it will resolve all facts
-  def resolve(fact)
+  def resolve(fact) # rubocop:disable Metrics/MethodLength
```

Comment:
instead of disabling cops in code we are leaving exceptions in rubocop_todo.yml

Reply:
@GSPatton removed :-)